app/import.go

```diff
@@ -180,10 +191,3 @@ func (a *App) importLine(line LineImportData, dryRun bool) *model.AppError {
 		return model.NewAppError("BulkImport", "app.import.import_line.unknown_line_type.error", map[string]interface{}{"Type": line.Type}, "", http.StatusBadRequest)
 	}
 }
-
-func (a *App) finalizeImport(dryRun bool) *model.AppError {
```

Comment:
Added @jespino to the review as I don't understand (or perhaps remember) how this function ended up being so pointless but still here.

Reply:
This was added to do certain calculations at the end of the export (channels last update at, and things like that), because the order of the posts doesn't need to be in the same order than the channel, but that was finally solve in other place, I think we can remove the `finalizeImport` without problem.